### ISSUE TITLE
Angular 10 compatibility

### ISSUE THREAD

**Issue by jcroldan** [lsqlabs/ngx-guided-tour#67]
Hi,

i'm trying to compile your library with angular vesion 10 and i get the flowing error:

```
Compiling TypeScript sources through ngc
ERROR: node_modules/ngx-guided-tour/lib/guided-tour.module.d.ts:3:23 - error TS2314: Generic type 'ModuleWithProviders<T>' requires 1 type argument(s).

3     static forRoot(): ModuleWithProviders;
                        ~~~~~~~~~~~~~~~~~~~

An unhandled exception occurred: node_modules/ngx-guided-tour/lib/guided-tour.module.d.ts:3:23 - error TS2314: Generic type 'ModuleWithProviders<T>' requires 1 type argument(s).

3     static forRoot(): ModuleWithProviders;
                        ~~~~~~~~~~~~~~~~~~~

See "C:\Users\jcroldan\AppData\Local\Temp\ng-bHA7Fi\angular-errors.log" for further details.
child_process.js:660
    throw err;
    ^

```

I have found at angular 10 documentation that it is necessary to change ModuleWithProviders to 
`ModuleWithProviders<NgxGuidedTourModule>`

I have forked the project and opened a PR https://github.com/lsqlabs/ngx-guided-tour/pull/66

Could you, please, review the PR and aprove if you consider it is correct?

Best regards.

**Comment by RobertCMusk** [comment]
@jcroldan I have released version`1.1.11` with your change. Thank you for your contribution! I apologize for how long it took us to get around to reviewing it.

**Comment by jcroldan** [comment]
Thanks to you and congrats for your excellent work.
